docs/source/Learn/bskPrinciples/bskPrinciples-11.rst

```diff
@@ -0,0 +1,245 @@
+.. _bskPrinciples-11:
+
+Advanced: Effector Module Branching
+===================================
+
+Module branching in Basilisk enables the attachment of compatible dynamic effectors on state
+effectors as an alternative to attaching them directly to the hub. For example, a gimballed thruster
+can be modeled as a ``thrusterDynamicEffector`` attached to a ``spinningBodyTwoDOFStateEffector``,
+or a robotic docking arm can be modeled as a ``constraintDynamicEffector`` attached to a 7-DOF
+revolute ``spinningBodyNDOFStateEffector``. Multiple dynamic effectors can be attached to a state
+effector as well, for example applying both drag and SRP to a solar panel.
+
+Further details on how to set up new state effectors and dynamic effector modules for branching can
+be found in :ref:`effectorBranching`. Integrated testing of effector branching is elaborated on in
+:ref:`test_effectorBranching_integrated`.
+
+Allowable Configurations
+------------------------
+The currently supported configurations of state effectors compatible as "parent" effectors and
+dynamic effectors compatible as "child" effectors is summarized in the table below. Green shows
+currently supported configurations, yellow shows configurations expected to be supported in the
+future, and red shows configurations not planned to be supported. Additionally, blue shows the
+special case of prescribed effector branching explained in :ref:`prescribedMotionStateEffector`.
+
+.. raw:: html
+
+    <style>
```

Comment:
table looks great in Chrome, not good in Safari.  This is a nice looking table, but also rather complex to maintain.  Can we go with something simpler and more cross-browser friendly?

Reply:
I pushed a commit that provides compatibility with Chrome, Edge and Safari.  Please integrate that commit with your this commit.